### PR TITLE
Fixed shortCut issue and added small UI improvements

### PR DESCRIPTION
**Pull request by Shmigolk**
Aligned pictures in Product information and changed navBar

image

### CODE REVIEW

File: src/components/ResponsiveAppBar.jsx [hunk 1]
```diff
@@ -126,10 +127,19 @@ const ResponsiveAppBar = () => {
               display: { xs: "none", md: "flex" },
               flexDirection: "row",
               alignItems: "center",
+              width: '100%',
+              justifyContent: 'space-between'
```
Comment (alexfauquette) [inline src/components/ResponsiveAppBar.jsx:131]: A hack to add some space is to set an empty `div` with flexGrowth = 1
```jsx
<Box sx={{flexGrowth: 1}}></Box>
```

This div will extend itself to fill all the remaining space

File: src/pages/questions/QuestionDisplay.jsx [hunk 2]
```diff
@@ -125,7 +149,7 @@ const QuestionDisplay = ({ question, answerQuestion, resetFilters }) => {
         justifyContent="center"
         spacing={2}
         sx={{ mb: 1 }}
-        onKeyDown={(event) => {
+        /*onKeyDown={(event) => {
```
Comment (alexfauquette) [inline src/pages/questions/QuestionDisplay.jsx:152]: You can delete it now

File: src/pages/questions/ProductInformation.jsx [hunk 3]
```diff
@@ -122,7 +123,7 @@ const ProductInformation = ({ question }) => {
         labelPlacement="end"
       />
       {!hideImages && productData?.images && (
-        <Stack spacing={2} direction="row" flexWrap="wrap">
+        <Stack direction="row" flexWrap="wrap" gap='1em'>
```
Comment (nobeeakon) [inline src/pages/questions/ProductInformation.jsx:126]: Possibly better to use [Grid](https://mui.com/material-ui/react-grid/)?

File: src/pages/questions/QuestionDisplay.jsx [hunk 4]
```diff
@@ -62,6 +62,30 @@ const QuestionDisplay = ({ question, answerQuestion, resetFilters }) => {
...
+            answerQuestion({ value: 0, insightId: question.insight_id });
+            break;
+          default:
+            break;
+        }
+      }}
+
+    window.addEventListener( 'keydown', handleShortCut)
+    return () =>  window.removeEventListener('keydown', handleShortCut)
+  }, [question, answerQuestion])
```
Comment (nobeeakon) [inline src/pages/questions/QuestionDisplay.jsx:87]: better to use question?.insight_id in the dependency array (as it is a string and  question is an object so will change more frequently). Also you may need to check if the value is not null at the begnning of the useEffect.

File: src/pages/questions/ProductInformation.jsx (hunk 3 above)
Comment (alexfauquette) [reply src/pages/questions/ProductInformation.jsx:126]: The idea was to fix image's width and let them wrap when needed, but using Grid with a number of images per row varying with breaking points sounds interesting

Comment (alexfauquette) [reply src/pages/questions/ProductInformation.jsx:126]: I merged because the main topic was fixing the shortcuts. I open an issue for using Grid

**Pull request merged**